tests/test_build.py

```diff
@@ -5,17 +5,25 @@
 and is more unit-test oriented.
 """
 
+from __future__ import annotations
+
 import json
 import os
 import sys
 from pathlib import Path
+from typing import TYPE_CHECKING
 
+import pytest
 from conda.common.compat import on_win
 
 from conda_build import api, build
+from conda_build.exceptions import CondaBuildUserError
 
 from .utils import get_noarch_python_meta, metadata_dir
 
+if TYPE_CHECKING:
```

Comment:
Does this conditional speed up the tests

Reply:
no, it makes the linter happy since we have `TCH` enabled, I don't see a strong argument for disabling this lint rule for tests